### ISSUE TITLE
Use case for consideration: SEEKs Playroom

### ISSUE THREAD

**Issue by kevinpschaaf** [PolymerLabs/playground-elements#97]
https://github.com/Polymer/lit-element/issues/1139#issuecomment-765697272 noted a desire to build a web component version of [SEEK's Playroom](https://seek-oss.github.io/seek-style-guide/playroom/). It occurs to me that it should be trivial to build a similar experience using the `playground-elements`.

This gets pretty far, although I haven't fully tested it: https://jsbin.com/kazaciy/edit?html,output. I stopped playing with it once I realized the workers need to be served from the same domain, so it needs to be done from a local npm install served together with the app itself.

The main thing I wanted to confirm was whether it's legal to have multiple `playground-preview` elements (of different sizes, in this case) pointing to the same project, and it appears to be fine? Assuming so, there's probably not much else playground-elements would need to change to build something like that experience.

@calebdwilliams Posting this here (as opposed to in the lit-element issue so we don't derail that) in case this sparks some inspiration and you wanted to run with it!  @aomarks Feel free to close the issue unless it suggests some features to consider.

**Comment by aomarks** [comment]
Yeah, multiple preview elements for the same project should work fine. They just listen for events from the `<playground-project>` element they are connected to. (You can also have multiple `<playground-file-editor>` elements pointed as the same project, each controlling different filenames).
